lib/lago/__init__.py

```diff
@@ -620,8 +643,8 @@ def _ova_to_spec(self, filename):
 
         if image_file is not None:
             disk_meta = {"root-partition": "/dev/sda1"}
-
-            disk_spec = [{"type": "file",
+            disk_spec = [{"type": "template",
+                          "template_type": "file",
```

Comment:
this is a backwards incompatible change, you pass from using a  file directly to using a template, I don't think that noone but you is currently using it so if it's ok for you it's ok for me, but keep in mind that it will not be allowed to be that simple to change like that in the future